src/style.css

```diff
@@ -0,0 +1,79 @@
+body {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: white;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.boxes {
+  flex: 1 0 33%;
+  height: 300px;
+  min-width: 300px;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
```

Comment:
remove

Reply:
it helps me to center the digits, why i have to remove this code?

Reply:
empty line removed